testing/xcb-util-xrm/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Maintainer: Jakub Skrzypnik <j.skrzypnik@openmailbox.org>
+pkgname=xcb-util-xrm
+pkgver=1.2
+pkgrel=0
+pkgdesc="Utility functions for the X resource manager"
+url="https://github.com/Airblader/xcb-util-xrm/"
+arch="all"
+license="MIT"
+makedepends="libxcb-dev xcb-util-dev util-macros bsd-compat-headers automake autoconf libx11-dev"
+source="https://github.com/Airblader/xcb-util-xrm/releases/download/v1.2/xcb-util-xrm-1.2.tar.gz"
```

Comment:
Please don’t hard-code version in source URL (it complicates bumping of abuild), use variable `$pkgver`.